hack/install.sh

```diff
@@ -11,13 +11,59 @@
 VERSION=${DAYTONA_SERVER_VERSION:-"latest"}
 BASE_URL=${DAYTONA_SERVER_DOWNLOAD_URL:-"https://download.daytona.io/daytona"}
 DESTINATION=${DAYTONA_PATH:-"/usr/local/bin"}
+CONFIRM_FLAG=false
+
+# Check for the -y flag
+for arg in "$@"; do
+  case $arg in
+    -y)
+    CONFIRM_FLAG=true
+    shift
+    ;;
+  esac
+done
 
 # Print error message to stderr and exit
 err() {
   echo "[$(date +'%Y-%m-%dT%H:%M:%S%z')]: $*" >&2
   exit 1
 }
 
+# Stop the Daytona server if it is running
+stop_daytona_server() {
+  if pgrep -x "daytona" > /dev/null; then
+    if [ "$CONFIRM_FLAG" = false ]; then
+      read -p "Daytona server is running. Do you want to stop it? (yes/no): " user_input < /dev/tty
+      case $user_input in
+        [Yy][Ee][Ss] )
+          CONFIRM_FLAG=true
+          ;;
+        [Nn][Oo] )
+          echo "Please stop the Daytona server manually and rerun the script."
+          exit 0
+          ;;
+        * )
+          echo "Invalid input. Please enter 'yes' or 'no'."
+          exit 1
+          ;;
+      esac
+    fi
+
+    if [ "$CONFIRM_FLAG" = true ]; then
+      echo "Attempting to stop the Daytona server..."
+      if daytona server stop; then
+        echo "Daytona server stopped."
+      else
+        echo "Attempting to kill the process..."
+        pkill -x "daytona"
+      fi
+    fi
```

Comment:
@ezhil56x lets just add a new line after all these inputs to make it a bit cleaner and we're ready to go. Nice work!

Reply:
Just one echo before exiting the if block should be enough

Reply:
Do you want me to add it after `pkill`?

Reply:
Just before the `fi` of the `if` block that checks the CONFIRM_FLAG. That will make all the output of that block separated from the next output below.

Reply:
@Tpuljak Is this fine
```
    if [ "$CONFIRM_FLAG" = true ]; then
      echo "Attempting to stop the Daytona server..."
      if daytona server stop; then
        continue
      else
        pkill -x "daytona"
      fi
      echo "Daytona server stopped."
    fi
```

Reply:
That's also great, but please add another newline at the end as well.

With this change, you would have
```
Daytona server stopped.
Installing Daytona ....
```

But I suggest we have
```
Daytona server stopped.

Installing Daytona ....
```